// @staffbase/widget-sdk/dist/index.js
"use strict";
var __defProp = Object.defineProperty;
var __getOwnPropDesc = Object.getOwnPropertyDescriptor;
var __getOwnPropNames = Object.getOwnPropertyNames;
var __hasOwnProp = Object.prototype.hasOwnProperty;
var __export = (target, all) => {
  for (var name in all)
    __defProp(target, name, { get: all[name], enumerable: true });
};
var __copyProps = (to, from, except, desc) => {
  if (from && typeof from === "object" || typeof from === "function") {
    for (let key of __getOwnPropNames(from))
      if (!__hasOwnProp.call(to, key) && key !== except)
        __defProp(to, key, { get: () => from[key], enumerable: !(desc = __getOwnPropDesc(from, key)) || desc.enumerable });
  }
  return to;
};
var __toCommonJS = (mod) => __copyProps(__defProp({}, "__esModule", { value: true }), mod);

// lib/index.ts
var lib_exports = {};
__export(lib_exports, {
  IntegrationNames: () => IntegrationNames,
  IntegrationStates: () => IntegrationStates
});
module.exports = __toCommonJS(lib_exports);

// lib/widget-api-types.ts
var IntegrationNames = /* @__PURE__ */ ((IntegrationNames2) => {
  IntegrationNames2["MS365"] = "ms365";
  IntegrationNames2["GoogleWorkspace"] = "googleWorkspace";
  IntegrationNames2["ServiceNow"] = "serviceNow";
  IntegrationNames2["Atlassian"] = "atlassian";
  return IntegrationNames2;
})(IntegrationNames || {});
var IntegrationStates = {
  UNAVAILABLE: "unavailable",
  AVAILABLE: "available",
  LOGGED_OUT: "loggedOut"
};
/*!
 * Copyright 2022, Staffbase GmbH and contributors.
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *     http://www.apache.org/licenses/LICENSE-2.0
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */
