// @staffbase/widget-sdk/dist/index.mjs
// lib/widget-api-types.ts
var IntegrationNames = /* @__PURE__ */ ((IntegrationNames2) => {
  IntegrationNames2["MS365"] = "ms365";
  IntegrationNames2["GoogleWorkspace"] = "googleWorkspace";
  IntegrationNames2["ServiceNow"] = "serviceNow";
  IntegrationNames2["Atlassian"] = "atlassian";
  return IntegrationNames2;
})(IntegrationNames || {});
var IntegrationStates = {
  UNAVAILABLE: "unavailable",
  AVAILABLE: "available",
  LOGGED_OUT: "loggedOut"
};
export {
  IntegrationNames,
  IntegrationStates
};
/*!
 * Copyright 2022, Staffbase GmbH and contributors.
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *     http://www.apache.org/licenses/LICENSE-2.0
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */
